estate/models/estate_model.py

```diff
@@ -0,0 +1,38 @@
+from odoo import models, fields
+from datetime import timedelta
```

Comment:
```suggestion
from datetime import timedelta

from odoo import models, fields
```
Generally, we make external imports first and then imports from odoo.
https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html